hwilib/devices/trezor.py

```diff
@@ -326,6 +326,8 @@ def _check_unlocked(self) -> None:
             self.client.ui.disallow_passphrase()
         if self.client.features.pin_protection and not self.client.features.unlocked:
             raise DeviceNotReadyError('{} is locked. Unlock by using \'promptpin\' and then \'sendpin\'.'.format(self.type))
+        if self.client.features.passphrase_protection and self.password is None:
```

Comment:
```suggestion
        if self.client.features.passphrase_protection and not self.client.features.unlocked:
```

Otherwise, `./hwi.py -t "trezor" -d "webusb:020:2:4" signtx ... ` doesn't work! This would also break the signing flow in Specter Desktop with a Model T right now.